### PR TITLE
Dockerfile improvements, ARM64 binaries

### PR DESCRIPTION
**Pull request by mrueg**
This change improves dockerfiles and allows to build conftest binaries for other architectures.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,33 +24,33 @@ RUN go test -v ./...
...
-RUN wget -O /usr/local/bin/kustomize https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64 && \
+RUN wget -O /usr/local/bin/kustomize "https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64" && \
     chmod +x /usr/local/bin/kustomize
 
 RUN go get -u cuelang.org/go/cmd/cue
 
 WORKDIR /examples
 
 ## RELEASE ##
-FROM alpine:latest
```
Comment (garethr) [inline Dockerfile:50]: If you could change this back for now. Happy to discuss in a separate issue but it introduces more maintenance overhead from my point of view.

**Comment by garethr** [comment]
Thanks  @mrueg. This looks great. I couple of small things:

* Definitely worth adding a mention of the ARM binaries to the installation docs https://github.com/open-policy-agent/conftest/blob/master/docs/install.md
* If you could sign the developer consent order (DCO). You can see the details here https://github.com/open-policy-agent/conftest/pull/347/checks?check_run_id=881699022

File: Dockerfile (hunk 1 above)
Comment (mrueg) [reply Dockerfile]: Sure, changed it back. In general latest makes it harder to reproduce a specific image, that's why I would suggest a proper tagged release.

**Comment by mrueg** [comment]
Thanks for the quick review, I've added a note to the docs and added my DCO signoff.

**Comment by garethr** [comment]
@mrueg thanks, looks good. We'll ship ARM binaries in the next release.

The tags only marginally help with reproducibility, as those are ephemeral as well. Using a SHA works, but without better tooling to help maintain it's painful. The tradeoff with latest are worth it for a project like Conftest at the moment.

**Pull request merged**